### PR TITLE
chore(chat): move report issue and request api key actions to epics (Issue #117)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

Move report issue and request api key actions to epics

Issues:

- #117 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1206.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1206.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1206.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/ReportIssueDialog.tsx [hunk 1]
```diff
@@ -72,6 +67,15 @@ export const ReportIssueDialog: FC<Props> = ({ isOpen, onClose }) => {
     [],
   );
 
+  useEffect(() => {
+    if (isSuccessfullySent) {
+      dispatch(ServiceActions.resetIsSuccessfullySent());
+
+      setTitle('');
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ReportIssueDialog.tsx:74]: Should we close the dialog after sending?

File: apps/chat/src/store/service/service.reducer.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      state,
+      _action: PayloadAction<{
+        access_scenario: string;
+        business_reason: string;
+        project_end: string;
+        project_id: string;
+        project_lead: string;
+        project_stream: string;
+        workload_pattern: string;
+      }>,
```
Comment (IlyaBondar) [inline apps/chat/src/store/service/service.reducer.ts]: Could you define a type and reuse it in all places instead, please

File: apps/chat/src/components/Chat/ReportIssueDialog.tsx (hunk 1 above)
Comment (Alexander-Kezik) [reply apps/chat/src/components/Chat/ReportIssueDialog.tsx:74]: It closes in handleSubmit method

File: apps/chat/src/types/request-api-key.ts [hunk 3]
```diff
@@ -1,10 +1,9 @@
 export interface RequestAPIKeyBody {
   project_id: string;
   project_stream: string;
-  project_lead: string;
   project_end: string;
   business_reason: string;
   workload_pattern: string;
   access_scenario: string;
-  requester_email: string;
```
Comment (IlyaBondar) [inline apps/chat/src/types/request-api-key.ts:9]: Why was this field deleted?

Comment (Alexander-Kezik) [reply apps/chat/src/types/request-api-key.ts:9]: This field is not present in the request api key dialog, and this type was not used anywhere, I used it now.

There was a type `Omit<RequestAPIKeyBody, 'requester_email'>`, but actually pure RequestAPIKeyBody was not used anywhere

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1206.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1206.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**